src/components/DailyStatistics/DailyStatisticsScreen.tsx

```diff
@@ -0,0 +1,266 @@
+import {
+  AlertModalContext,
+  systemAlertProps,
+  ERROR_MESSAGE
+} from "../../context/alert";
+import { navigateHome } from "../../common/navigation";
+import { NavigationProps } from "../../types";
+
+interface StatisticsContext {
+  totalCount: number | null;
+  currentTimestamp: number;
+  lastTransactionTime: number | null;
+  transactionHistory: { name: string; category: string; quantity: number }[];
+  setTotalCount: (totalCount: number | null) => void;
+  setCurrentTimestamp: (currentTimestamp: number) => void;
+  setLastTransactionTime: (lastTransactionTime: number) => void;
+  setTransactionHistory: (
+    transactionHistory: { name: string; category: string; quantity: number }[]
+  ) => void;
+  clearStatistics: () => void;
+}
+
+export const StatisticsContext = createContext<StatisticsContext>({
+  totalCount: null,
+  currentTimestamp: Date.now(),
+  lastTransactionTime: null,
+  transactionHistory: [],
+  setTotalCount: () => null,
+  setCurrentTimestamp: () => null,
+  setLastTransactionTime: () => null,
+  setTransactionHistory: () => null,
+  clearStatistics: () => null
+});
+
+export const StatisticsContextProvider: FunctionComponent = ({ children }) => {
+  const [totalCount, setTotalCount] = useState<StatisticsContext["totalCount"]>(
+    null
+  );
+  const [currentTimestamp, setCurrentTimestamp] = useState<
+    StatisticsContext["currentTimestamp"]
+  >(Date.now());
+  const [lastTransactionTime, setLastTransactionTime] = useState<
+    StatisticsContext["lastTransactionTime"]
+  >(null);
+  const [transactionHistory, setTransactionHistory] = useState<
+    StatisticsContext["transactionHistory"]
+  >([]);
+
+  const clearStatistics: StatisticsContext["clearStatistics"] = useCallback(() => {
+    setTotalCount(null);
+    setCurrentTimestamp(Date.now());
+    setLastTransactionTime(null);
+    setTransactionHistory([]);
+  }, []);
+
+  return (
+    <StatisticsContext.Provider
+      value={{
+        totalCount,
+        currentTimestamp,
+        lastTransactionTime,
+        transactionHistory,
+        setTotalCount,
+        setCurrentTimestamp,
+        setLastTransactionTime,
+        setTransactionHistory,
+        clearStatistics
+      }}
+    >
+      {children}
+    </StatisticsContext.Provider>
+  );
+};
+
+const styles = StyleSheet.create({
+  content: {
+    padding: size(2),
+    paddingVertical: size(6),
+    height: "100%",
+    width: 512,
+    maxWidth: "100%"
+  },
+  headerText: {
+    marginBottom: size(4)
+  },
+  bannerWrapper: {
+    marginBottom: size(1.5)
+  },
+  categoryName: {
+    fontFamily: "brand-bold",
+    fontSize: fontSize(1),
+    marginBottom: size(3)
+  }
+});
+
+const DailyStatisticsScreen: FunctionComponent<NavigationProps> = ({
+  navigation
+}) => {
+  useEffect(() => {
+    Sentry.addBreadcrumb({
+      category: "navigation",
+      message: "DailyStatisticsScreen"
+    });
+  }, []);
+
+  const messageContent = useContext(ImportantMessageContentContext);
```

Comment:
This is the yellow banner! right now it only includes the message to restart the app to apply updates.

Original intention is to provide a way for us to push down important messages, but so far no use case other than updates.

Reply:
Oic, thanks for explaining! 

@dalsontws if this is the template for many screens, maybe we can considered having a template component? Rather than have to remember to add the update banner for each. Template component can have all the things like the SupplyAlly logo, the top half background, card in the middle, update banner, things like that.

Reply:
thats a good idea! i think we can do that in another story, together with the `Need help` issue, perhaps thinking of moving the need help into the drawer!